jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -384,15 +389,26 @@ private void pushContainerConfiguration() {
     ProgressEventDispatcher.Factory childProgressDispatcherFactory =
         Verify.verifyNotNull(rootProgressDispatcher).newChildProducer();
 
-    results.containerConfigurationPushResult =
+    results.containerConfigurationPushResults =
         executorService.submit(
-            () ->
-                new PushContainerConfigurationStep(
-                        buildContext,
-                        childProgressDispatcherFactory,
-                        results.targetRegistryClient.get(),
-                        results.builtImages.get().get(0).get())
-                    .call());
+            () -> {
+              Map<Image, Future<BlobDescriptor>> containerConfigurationPushResults =
```

Comment:
Let's rename this to `builtImagesAndContainerConfigurationPushResults`.

Using a `Map` is convenient, but compared to using a dedicated class, it's difficult to know what the keys and the values mean and what is the relation between them without documentation or descriptive naming. For a short time, I had to go over the code to see if `Image` in the map is a base image or a built image. We may keep the current name, but in that case, we should add a clear Javadoc to the field.

Reply:
I think its a good idea change the name to ` builtImagesAndContainerConfigurationPushResults `.I will also change image to `baseImage` or to `builtImage`

Reply:
As mentioned in the other comment, how about
```
Map<Image, Future<BlobDescriptor>> pushResults =
```
? (I think `buildImage` in `for (Future<Image> builtImage` : ...` is good though.)